app/policies/chat_channel_policy.rb

```diff
@@ -51,6 +51,10 @@ def joining_invitation_response?
     record.present?
   end
 
+  def create_channel?
+    record.present? && user.has_role?(:mod_relations_admin)
```

Comment:
There is only 1 user with this role. Maybe we should use `:tag_moderator`, there are 274 of them and we have a tag mod newsletter where we can inform them about this new feature so they actually test it. /cc @michael-tharrington